dpti/gdi.py

```
@@ -241,8 +241,10 @@ def make_dpdt(
         else:
             raise RuntimeError("invalid inte_dir " + inte_dir)
         assert min_idx >= 0
-        conf_0 = os.path.join("database", "task.%06d" % min_idx, "0", "out.lmp")
-        conf_1 = os.path.join("database", "task.%06d" % min_idx, "1", "out.lmp")
+        # conf_0 = os.path.join("database", "task.%06d" % min_idx, "0", "out.lmp")
+        # conf_1 = os.path.join("database", "task.%06d" % min_idx, "1", "out.lmp")
+        conf_0 = os.path.join("database", "task.%06d" % min_idx, "0", "final.lmp")
```

Ruff (0.11.9)

Use format specifiers instead of percent format

(UP031)